lib/routes/api/speech-credentials.js

```diff
@@ -110,24 +116,29 @@ router.get('/', async(req, res) => {
     res.status(200).json(creds.map((c) => {
       const {credential, ...obj} = c;
       if ('google' === obj.vendor) {
-        obj.service_key = JSON.parse(decrypt(credential));
+        const o = JSON.parse(decrypt(credential));
+        const obsecured = {
+          client_email: o.client_email,
+          private_key: obscureKey(o.private_key)
+        };
+        obj.service_key = obsecured;
       }
       else if ('aws' === obj.vendor) {
         const o = JSON.parse(decrypt(credential));
         obj.access_key_id = o.access_key_id;
-        obj.secret_access_key = o.secret_access_key;
+        obj.secret_access_key = obscureKey(o.secret_access_key);
         obj.aws_region = o.aws_region;
         logger.info({obj, o}, 'retrieving aws speech credential');
       }
       else if ('microsoft' === obj.vendor) {
         const o = JSON.parse(decrypt(credential));
-        obj.api_key = o.api_key;
+        obj.api_key = obscureKey(o.api_key);
         obj.region = o.region;
         logger.info({obj, o}, 'retrieving azure speech credential');
       }
       else if ('wellsaid' === obj.vendor) {
         const o = JSON.parse(decrypt(credential));
-        obj.api_key = o.api_key;
+        obj.api_key = obscureKey(o.api_key);
```

Comment:
Have you validated why the actual api response doesn’t send this field to the client? We should make sure that fix is also part of this PR 🤔

Reply:
Yea, it is pretty silly. There is no WellSaid condition for the single fetch, but there is one for fetch multiple. Easy and quick fix.

Reply:
Awesome